department-of-veterans-affairs/vets-website
# Arf 83871 Form 21a Step 1

## Summary

- Added step pages to align with the 21A V2 Designs flow (Note: there are some aspects that align with V3 Designs that were confirmed with design as mostly set - if you do not see the UI element in V2 check the V3 tab.)
- [Designs](https://www.figma.com/design/2afIGOMII0uRI5ck1dWo1w/Form-21a---Apply-for-Accreditation-(CA-%26-Attorneys)?node-id=262-22141&t=JXSWf514L6nfbf8n-4)
- Added e2e testing
- Accredited Representative Facing Team

## Related issue(s)

- [Merge Form 21a Step 1 (Personal Information)](https://app.zenhub.com/workspaces/accredited-representative-facing-team-65453a97a9cc36069a2ad1d6/issues/gh/department-of-veterans-affairs/va.gov-team/83871)
- [[ARF] Digital 21a](https://app.zenhub.com/workspaces/accredited-representative-facing-team-65453a97a9cc36069a2ad1d6/issues/gh/department-of-veterans-affairs/va.gov-team/83627)

## Testing done

- Navigate from Introduction page through all 7 steps especially step 1 to review and submit
- Ensure the pages align with [designs](https://www.figma.com/design/2afIGOMII0uRI5ck1dWo1w/Form-21a---Apply-for-Accreditation-(CA-%26-Attorneys)?node-id=99-3435&t=W5LREYSL0uiLC764-4)

## Screenshots

### Before - these pages did not exist

### After
<img width="1086" alt="Screenshot 2024-06-14 at 9 30 31 AM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/22990386/bc0e1eb6-7123-4827-bed7-f134e8588927">
<img width="1087" alt="Screenshot 2024-06-14 at 9 31 01 AM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/22990386/f8fac237-3643-449e-8927-b9111fc2da9e">
<img width="1089" alt="Screenshot 2024-06-14 at 9 31 19 AM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/22990386/4fa01db7-561a-411a-a3ed-68d1c4f51ba9">
<img width="1092" alt="Screenshot 2024-06-14 at 9 31 57 AM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/22990386/82707f1a-374e-4667-be8e-bb7ecd46c567">
<img width="1090" alt="Screenshot 2024-06-14 at 9 32 22 AM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/22990386/ff8b7217-942a-4e21-be64-96a9ca1209e1">
<img width="1086" alt="Screenshot 2024-06-14 at 9 32 49 AM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/22990386/e4729d21-04db-4e58-a368-5f1384fd3dd7">
<img width="1080" alt="Screenshot 2024-06-14 at 9 33 04 AM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/22990386/3b09f34c-8d6e-4cb0-9375-03605d902815">
<img width="1030" alt="Screenshot 2024-06-14 at 9 39 25 AM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/22990386/1fba39d5-a2ee-4e7f-a0f6-2ee100995360">
<img width="1029" alt="Screenshot 2024-06-14 at 9 39 55 AM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/22990386/4b9cad67-0a61-4d18-b7f7-bc9777b67893">
<img width="1034" alt="Screenshot 2024-06-14 at 9 40 23 AM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/22990386/1e83a412-3307-4d90-8379-93924d3815d4">
Note: this does not include the address below the radio label custom UI as this design element could be in flux and is not a pattern in the VA Forms Library
<img width="1028" alt="Screenshot 2024-06-14 at 9 40 47 AM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/22990386/09b9213e-32e7-48a1-a4c0-8ad4e988af7f">


## Acceptance criteria

### Quality Assurance & Testing
